keyboards/wsk/gothic70/keymaps/default/keymap.c

```diff
@@ -0,0 +1,83 @@
+
+#include QMK_KEYBOARD_H
+
+
```

Comment:
```suggestion
// Defines names for use in layer keycodes and the keymap
enum layer_names { 
    _BASE,
    _FN, 
    _MEDIA };
```

Using an enum for layernames can make the keyboards more readable :)